### PR TITLE
BringUp Validus

### PR DESCRIPTION
**Pull request by lonewandererhari123**

### CODE REVIEW

**Comment by vlw** [comment]
@lonewandererhari123 CR -2

File: validus.mk [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 PRODUCT_BRAND := samsung
 PRODUCT_MODEL := SM-J500F
 PRODUCT_MANUFACTURER := samsung
 PRODUCT_CHARACTERISTICS := phone
 
 PRODUCT_GMS_CLIENTID_BASE := android-samsung
+
+# Device Maintainers
+PRODUCT_BUILD_PROP_OVERRIDES += \
+    DEVICE_MAINTAINERS="tioguda"
```
Comment (vlw) [inline validus.mk:18]: maintainer is @vince2678

File: vendorsetup.sh [hunk 2]
```diff
@@ -1,3 +1 @@
-for i in eng userdebug user; do
-	add_lunch_combo lineage_j5lte-${i}
-done
+add_lunch_combo validus_j5lte-userdebug
```
Comment (vlw) [inline vendorsetup.sh:1]: doesn't think that it is good idea

File: validus.mk (hunk 1 above)
Comment (vlw) [reply validus.mk:18]: also branch cm-14.1 isn't for "validus"

**Comment by minz1** [comment]
@lonewandererhari123 Can you please close this PR? This contributes absolutely nothing, Keep your rebrands into your own personal fork.

**Comment by vzvikaramba** [comment]
1. We don't merge PRs (anymore). Everything goes through review (http://review.msm8916.com) 

2. This is not the appropriate place for rom-specific changes. Those can be kept in forked trees where it's appropriate.

**Pull request closed without merging**